cmd/pop/cli/get.go

```diff
@@ -39,7 +39,7 @@ data to disk. Adding a miner flag will fallback to miner if content is not avail
 		fs.BoolVar(&getArgs.verbose, "verbose", false, "print the state transitions")
 		fs.StringVar(&getArgs.miner, "miner", "", "ask storage miner and use as fallback if network does not have the content")
 		fs.StringVar(&getArgs.strategy, "strategy", "SelectFirst", "strategy for selecting offers from providers")
-		fs.Int64Var(&getArgs.maxppb, "maxppb", 5, "max price per byte")
+		fs.Int64Var(&getArgs.maxppb, "maxppb", 0, "max price per byte")
```

Comment:
We set a default PPB value at 0 in the Get(), so that when received by popn, if PPB is at 0, it will use the default value set when initialised the node.